src/lightstep-opentelemetry-launcher-node.ts

```diff
@@ -31,19 +36,20 @@ const PROPAGATOR_LOOKUP_MAP: {
   [key: string]:
     | typeof B3Propagator
     | typeof HttpTraceContext
-    | typeof HttpCorrelationContext;
+    | typeof HttpBaggage;
 } = {
   b3: B3Propagator,
   b3single: B3Propagator,
   tracecontext: HttpTraceContext,
-  baggage: HttpCorrelationContext,
+  baggage: HttpBaggage,
 };
 
 /** Default values for LightstepNodeSDKConfiguration */
 const LS_DEFAULTS: Partial<types.LightstepNodeSDKConfiguration> = {
   spanEndpoint: 'https://ingest.lightstep.com:443/api/v2/otel/trace',
-  metricEndpoint: 'https://ingest.lightstep.com:443/metrics',
+  metricEndpoint: 'https://ingest.lightstep.com/metrics/otlp/v0.5',
   propagators: PROPAGATION_FORMATS.B3,
+  metricsHostEnabled: 'true',
```

Comment:
Is it possible to make this a boolean`true`, rather than a string?

Reply:
> Is it possible to make this a boolean`true`, rather than a string?

this value might come from env variable which can only be a string also `LightstepConfigType` is declared to have string only, without better parsing and config validation (as in otel) I can't declare it as boolean, but string only.

Reply:
I think having `"true"` as a string is going to be error prone for users. What would it take for us to improve the configuration so that we can selectively convert environment values to boolean?

Reply:
Well basically the metrics are enabled by default so user doesn't need to do anything. IF the user wants to disable metrics you can do basically `metricsHostEnabled: "blabla"` and it will be disabled you can also set it to `metricsHostEnabled: false`  etc.. SO there will never be a situation when user want to enable this metrics as it is enable by default so you can only disable this. And you can achieve that by setting this value to anything but "true"

Reply:
and you don't have any other values then strings when setting this via environment instead of config ......

Reply:
That makes sense. Things work as expected, even though the implementation isn't ideal. We can look to clean that up later. Thanks for clearing that up.